src/gmail-sync-worker.js

```diff
@@ -0,0 +1,47 @@
+const {
+  GmailEmailSource,
+  getSimplifiedMessage
+} = require('./services/gmail-email-source');
+const { saveRequest, Request } = require('./airtable');
+const source = new GmailEmailSource(process.env.GMAIL_LABEL || 'incoming');
+
+const interval = parseInt(process.env.GMAIL_SYNC_INTERVAL || 5000);
+
+/**
+ * Wakes up every $GMAIL_SYNC_INTERVAL and polls gmail for new messages in $GMAIL_LABEL.
+ * It sends matching messages to airtable as Requests and then adds a label to them so that they aren't processed twice.
+ */
+async function syncGmail() {
+  try {
+    console.info('Spinning up gmail sync');
+    const newMessages = await source.poll();
+
+    console.info(`Found ${newMessages.length} to send to airtable`);
+    addedMessages = [];
+    for (const fullMessage of newMessages) {
+      const message = getSimplifiedMessage(fullMessage);
+      const request = new Request({
+        emailAddress: message.from,
+        textOrVoice: 'text',
```

Comment:
I think maybe this field in Airtable should be turned into source or something?

Reply:
agreed @mab253

Reply:
Alex I'd say you can just go for it. lmk if you need access. only thing to check on is if it's used in the twilio repo at all

Reply:
It is used in twilio so I feel like we can leave email as text for the time being (we can tell they are emails if we need to fix the data later) and when we get a better feel for the kinds of sources that will be generating requests think about a source column instead.